samcli/local/init/templates/cookiecutter-aws-sam-hello-java-gradle/{{cookiecutter.project_name}}/HelloWorldFunction/build.gradle

```diff
@@ -7,17 +7,6 @@ repositories {
 }
 
 dependencies {
-    implementation 'software.amazon.awssdk:annotations:2.1.0'
-	compile (
-        'com.amazonaws:aws-lambda-java-core:1.1.0'
-    )
-    testCompile 'junit:junit:4.12'
+    implementation 'com.amazonaws:aws-lambda-java-core:1.2.0'
```

Comment:
is "implementation 'software.amazon.awssdk:annotations:2.1.0" not required at all?

Reply:
It is not used in the template source code. I can build and deploy the application with no problem removing this.